roles/idrac_bios/molecule/clear_pending_attributes/prepare.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Converge file to update attributes with apply time as Immediate
```

Comment:
Update the name, it's prepare.yml not converge.yml, check other places too

Reply:
Updated.